### PR TITLE
[prerender] run worker init script sync

### PR DESCRIPTION
**Pull request by stu-zumper**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -54,6 +54,11 @@ exports = module.exports = function(options) {
             workAround(cluster.fork());
         });
     } else {
+        if (process.env.PRERENDER_WORKER_INIT_SCRIPT) {
+          runsync = require('runsync');
```
Comment (Woodya) [inline lib/index.js]: global var problem.
better to simply inline it:
```js
require('runsync').execFile(process.env.PRERENDER_WORKER_INIT_SCRIPT);
```
or to have a single module scope var at the top.

File: package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "lodash": "^3.1.0",
     "phantom": "^0.7.2",
     "phantomjs": "^1.9.15",
-    "tree-kill": "0.0.6"
+    "tree-kill": "0.0.6",
+    "runsync": "0.1.8"
```
Comment (Woodya) [inline package.json]: update our license spreadsheet.

File: lib/index.js (hunk 1 above)
Comment (Woodya) [inline lib/index.js]: nit: rename PRERENDER_WORKER_INIT_CMD, might not be a script..

Comment (stu-zumper) [inline lib/index.js]: Inlined.
Renamed.

File: package.json (hunk 2 above)
Comment (stu-zumper) [inline package.json]: Updated.

**Comment by Woodya** [comment]
squash n go.

**Pull request merged**